pkgs/applications/audio/spotifyd/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkgconfig, openssl
+, withALSA ? true, alsaLib ? null
+, withPulseAudio ? false, libpulseaudio ? null
+, withPortAudio ? false, portaudio ? null
+}:
+
+rustPlatform.buildRustPackage rec {
+  name = "spotifyd-${version}";
+  version = "0.2.4";
+
+  src = fetchFromGitHub {
+    owner = "Spotifyd";
+    repo = "spotifyd";
+    rev = "v${version}";
+    sha256 = "0nmi33wvagmasxb59jwhlgdqlh8gfmqgwib16k1wx885qhx9yg30";
+  };
+
+  # buildRustPackage currently seems to have some problems with [replace]
+  # directives in Cargo files (see #30742 and #47172), so don't use the
+  # replacement.
+  cargoPatches = [ ./noavxcrypto.patch ];
```

Comment:
spotifyd 0.2.11 was released and doesn't required this https://github.com/marsam/nixpkgs/commit/179aaa85c43